web/src/javascript/app.js

```diff
@@ -8,7 +8,11 @@ let currentState;
 async function updateLoop() {
   currentState = await fetchState(0);
   drawGame(currentState, gameCanvas);
+}
+
+async function updateLeaderBoard() {

```

Comment:
I might be wrong, but I think you don't need the `async` here. You need it when you do an await in the function.